app/views/liquids/_open_graph.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="crayons-card p-5">
```

Comment:
So, I am experiencing quite a bit of WONKY behavior in the UI. In a nutshell, the app seems to be injecting `<br>` randomly into my code, and also rending one of my paragraphs in a `highlight` block. 😵‍💫

Here is a Loom where I explain the wonkyness: https://www.loom.com/share/67764b8712674bc2b375beb54ac4525e

HELP! 🙏🏾 

@aitchiss

Reply:
can you try editing one line in your [config/environments/development.rb](https://github.com/forem/forem/blob/main/config/environments/development.rb#L73) file

```
  # Annotate rendered view with file names.
  config.action_view.annotate_rendered_view_with_filenames = true
```

Set this to false, restart the server, and check again? I _think_ this is the same problem we've seen with the markdown processor in development only (this is false in production already) leaving those html comments inside a `<p>` tag after some embeds.

Reply:
@djuber This fixed it! Thank you so much!